.gitignore

```diff
@@ -1,2 +1,3 @@
 /vendor
 /coverage.txt
+/.idea
```

Comment:
You should not add .idea folder to shared .gitignore. Configure globally your Git to ignore IDE specific files and stop spreading this .idea stuff across every repository you touch. Put this line to your ~/.config/git/ignore file. See more info in [gitignore(5)][man gitignore.5] manual page.

 [man gitignore.5]: https://git-scm.com/docs/gitignore